### PR TITLE
Add Heetch

### PR DESCRIPTION
**Pull request by jhchabran**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

--- 

Hello, I've added the company I'm working for, we're about 80 engineers working remotely :)

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @jhchabran, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 4eb9136).  See below for details:

```
heetch.md: Invalid section: "Remote Status".  Expected one of: ["Company blurb","Company size","Remote status","Region","Company technologies","Office locations","How to apply"]

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @jhchabran, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 1de2485). :heavy_check_mark:

File: company-profiles/heetch.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Heetch
+
+## Company blurb
+
+Heetch is mobility solution operating across Europe and Africa. It aims to provide a profitable and comfortable experience for
+both drivers and passengers.
```
Comment (dougaitken) [inline company-profiles/heetch.md]: looks like there is a random line break here?

File: company-profiles/heetch.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+The engineering division of the company is fully distributed and we work remotely since day 1. We're scattered all around Europe and a few other countries with a similar timezone.
+
+## Company size
+
+280
+
+## Remote status
+
+Heetch is 100% remote in the engineering division. Product division is remote as well, but people must be in a city where
+we're operating, so they can meet our users.
```
Comment (dougaitken) [inline company-profiles/heetch.md]: Again, another random line break in the middle here

File: company-profiles/heetch.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Region
+
+Heetch employs all across Europe and countries with similar timezones, +/- 1h.
```
Comment (dougaitken) [inline company-profiles/heetch.md]: `+/- 1h` of where? 

Do you mean `+/- 1h of CET` ( Central European Standard Time )?

**Comment by jhchabran** [comment]
Should be better know, thanks for catching the missing timezone 🙏  

I rebased things so the log is kept clean.

**Comment by dougaitken** [comment]
Thanks @jhchabran - sorry about the delay merging this.

**Pull request merged**